app/models/manageiq/providers/ansible_runner_workflow.rb

```diff
@@ -91,6 +91,15 @@ def post_execute
 
   protected
 
+  def route_signal(*args, deliver_on: nil)
```

Comment:
I know you and Jason are pretty "anti-comments" for reasons seemingly related to types of grains\*, but I think it makes sense to at least document the reasoning for this method here.  I agree that the code is relatively straight forward to understand, but the rational as to why this needs to exist really could use some context for future devs.


\* This was a bad "Basmati Rice" joke

Reply:
Yeah, that makes sense.